client/packages/invoices/src/Returns/api/hooks/document/useOutboundDeleteRows.ts

```diff
@@ -2,13 +2,13 @@ import {
   useTableStore,
   useTranslation,
   useQueryClient,
-  InvoiceNodeStatus,
   useDeleteConfirmation,
   useUrlQueryParams,
   useMutation,
 } from '@openmsupply-client/common';
 import { useOutbounds } from './useOutbounds';
 import { useReturnsApi } from '../utils/useReturnsApi';
+import { canDeleteOutboundReturn } from 'packages/invoices/src/utils';
```

Comment:
Can you change the import please - importing from 'packages/' like this will break the jest tests

```suggestion
import { canDeleteOutboundReturn } from '../../../../utils';
```